tests/scripts/test_models.py

```diff
@@ -70,7 +75,15 @@ def test_inheritance():
     assert isinstance(ExtendedInheritedUnitTestModel.fields['integer'], leapp.models.fields.Integer)
     assert isinstance(ExtendedInheritedUnitTestModel.fields['strings'], leapp.models.fields.List)
 
-    assert len(InheritedUnitTestModel.fields) == 2
+    assert len(InheritedUnitTestModel.fields) == 3
     assert isinstance(InheritedUnitTestModel.fields['integer'], leapp.models.fields.Integer)
     assert isinstance(InheritedUnitTestModel.fields['strings'], leapp.models.fields.List)
     assert InheritedUnitTestModel.fields is not UnitTestModel.fields
+
+
+def test_inheritance_list_field_not_shared():
+    model1 = InheritedUnitTestModel()
+    model2 = AnotherInheritedUnitTestModel()
```

Comment:
I'd also extend the tests to ensure that they test for identity of the items fields:
```suggestion
    model2 = AnotherInheritedUnitTestModel()
    model3 = InheritedUnitTestModel()
    model4 = AnotherInheritedUnitTestModel()
    assert model3.items is not model1.items
    assert model4.items is not model2.items
    assert model1.items is not model2.items
    assert model3.items is not model4.items
```